examples/light-switch-app/silabs/efr32/README.md

```diff
@@ -118,7 +118,7 @@ Silicon Labs platform.
           $ cd ~/connectedhomeip/examples/light-switch-app/silabs/efr32
           $ git submodule update --init
           $ source third_party/connectedhomeip/scripts/activate.sh
-          $ export EFR32_BOARD=BRD4161A
+          $ export EFR32_BOARD=BRD4187C
```

Comment:
While you are in those files `EFR32_BOARD` has been renamed to `SILABS_BOARD`. Can you update that too
```suggestion
          $ export SILABS_BOARD=BRD4187C
```